extensions/wikia/AdEngine/js/provider/btfBlocker.js

```diff
@@ -24,11 +24,12 @@ define('ext.wikia.adEngine.provider.btfBlocker', [
 	function isBTFDisabledByCreative() {
 		return win.ads.runtime.disableBtf;
 	}
+	var btfQueue = [],
+		btfQueueStarted = false,
```

Comment:
Should we clear this state on consecutive pvs on mobile-wiki?

Reply:
I believe we already do it: https://github.com/Wikia/app/pull/14650/files#diff-3ede2fc47081feff0bafc6b748c2cf03R37